src/main/public/js/select-toggled-content.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function () {
+  $('.timeline-row .form-control-select').on('change', function () {
+
+    var $content = $(this).next()
+
+    if (this.value) {
+      $content
+        .removeClass('js-hidden')
+        .attr('aria-hidden', 'false')
+        .attr('aria-expanded', 'false')
+      $content
+        .find('textarea')
+        .val('')
+      $content
+        .find('.evidence-message')
+        .addClass('visually-hidden')
+      $content
+        .find('.evidence-message-' + this.value)
+        .removeClass('visually-hidden')
+    } else {
+      $content
+        .addClass('js-hidden')
+        .attr('aria-hidden', 'true')
+        .attr('aria-expanded', 'false')
+      $content
+        .find('.evidence-message')
```

Comment:
The same here, that could be extracted outside `if-else` statement.

Reply:
done